landing-strip/core/src/main/java/com/novoda/landingstrip/Attributes.java

```diff
@@ -0,0 +1,87 @@
+package com.novoda.landingstrip;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.support.annotation.ColorRes;
+import android.support.annotation.LayoutRes;
+import android.util.AttributeSet;
+
+class Attributes {
+
+    @ColorRes
+    private static final int DEFAULT_INDICATOR_COLOUR = android.R.color.white;
+    private static final int DEFAULT_INDICATOR_HEIGHT = 5;
```

Comment:
this should be `@DimenRes` and point to a dimen - if it's default, it should be default DPs not px